lambdatimes/src/components/Content/Content.js

```diff
@@ -48,7 +65,10 @@ export default class Content extends Component {
           `selectedTab` that includes the currently selected tab
           and `selectTabHandler` that includes the function to change the selected tab
         */}
-        <Tabs tabs={this.state.tabs} />
+        <Tabs 
+        selectedTab={this.state.selectedTab}
+        selectTabHandler={this.state.changeSelected}
```

Comment:
Here, the `changeSelected` function isn't inside state so we would reference it by `this.changeSelected` instead.